doc/source/geometric_manipulations.rst

```diff
@@ -1,92 +1,74 @@
 Geometric Manipulations
 ========================
 
+*geopandas* makes available all the tools for geometric manipulations in the `*shapely* library <http://toblerity.org/shapely/manual.html>`_. 
 
+Note that documentation for all set-theoretic tools for creating new shapes using the relationship between two different spatial datasets -- like creating intersections, or differences -- can be found on the :doc:`set operations </set_operations>` page.
```

Comment:
`` :doc:`set operations </set_operations>` `` -> I don't think the slash is needed?